src/components/Table.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import './Table.css';
+
+const Row = ({ location }) => {
+  return (
+    <tr className="tableListing">
+      <td className="tableTitle">{location.title}</td>
+      <td className="tableDescription">{location.description}</td>
+      <td className="tableLink">
+        {location.pageid ? (
+          <a
+            href={`https://en.wikipedia.org/?curid=${location.pageid}`}
+            target="_blank"
+            external="true"
```

Comment:
I think we decided that external="true" is copied from something else. Because it isn't an attribute for an <a> tag.